test/local_integration_test.py

```diff
@@ -343,7 +343,7 @@ def _get_entities_by_project(self, entity_type, project_short_name):
         return entities
 
 
-class OpenAPIIntegrationTest(unittest.TestCase):
+class OpenAPIIntegrationTest(AlwaysTearDownTestCase):
```

Comment:
I wouldn't even do this one …

Reply:
My thought was to just always use `AlwaysTearDownTestCase`. It seemed to me like the best practice since anyone could cancel any test at any time and we don't want to risk leaving around any resources.

I'm curious why you disagree.

Reply:
We don't know which of the existing tearDown methods satisfy the requirement of not failing after a partial setup. Did that requirement disappear?

We should extend this to all tests at some point. Feel free to create a ticket. But we are stirring up the guts of unittest with this so I just want to introduce this slowly and cautiously.

Note that cancelling the `test` stage in the Gitlab pipeline crashes the runner. I wonder if this would help with that. But again, let's move slowly unless I'm mistaken about the requirement.